corehq/apps/app_manager/suite_xml/sections/details.py

```diff
@@ -390,8 +390,12 @@ def _get_relevant_expression(module, in_search):
     def _get_auto_launch_expression(module, in_search):
         allow_auto_launch = toggles.USH_CASE_CLAIM_UPDATES.enabled(module.get_app().domain) and not in_search
         auto_launch_expression = "false()"
-        if allow_auto_launch and module.search_config.auto_launch:
-            auto_launch_expression = XPath(AUTO_LAUNCH_EXPRESSION)
+        if allow_auto_launch:
+            if module.is_multi_select():
+                # AUTO_LAUNCH_EXPRESSION is incompatible with multi select case lists - See USH-1870
+                auto_launch_expression = "true()"
```

Comment:
shouldn't this only be `true()` if `module.search_config.auto_launch` is True? Otherwise we're always going to auto-launch regardless of the workflow configuration.